openlabcmd/openlabcmd/zk.py

```diff
@@ -35,8 +35,22 @@ def __init__(self, config=None):
                                          "object.")
         self._last_retry_log = 0
 
-    def _bytesToDict(self, data):
-        return json.loads(data.decode('utf8'))
+    def _bytesToDict(self, node_data, include_created_at=False,
+                     include_updated_at=False):
+        """ Convert zookeeper node data into dict."""
+        node_dict = node_data[0]
```

Comment:
node_data is a list and length always is 1 ?

Reply:
node_data  is a list, length is always 2.

Reply:
got, thx